### PR TITLE
Iam change check

### PR DESCRIPTION
**Pull request by s-block**
Check for changes in IAM roles

Check airflow dags repo pep8

### CODE REVIEW

File: airflow.yaml [hunk 1]
```diff
@@ -149,7 +159,196 @@ jobs:
...
+                policies=$(echo '[]' | jq .)
+                cat dag-images/images.txt | while read line
+                do
+                  id=$(docker create $line)
+                  docker cp $id:deploy.json - > deploy.json
+                  docker cp $id:iam_policy.json - > iam_policy.json
+                  docker rm -v $id
+                  role=$(cat deploy.json | jq -r '.role_name')
+                  policy=$(cat iam_policy.json | jq .)
+                  $policies=$(exho $policies | jq --arg policy $policy --arg role $role '. += [{role: $role:, policy: $policy}]')
```
Comment (davidread) [inline airflow.yaml]: `exho`?

Comment (s-block) [reply airflow.yaml]: Good spot

**Review by r4vi — APPROVED** [review]
testing this must've been painful

**Comment by s-block** [comment]
> testing this must've been painful

very

**Pull request merged**